hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -377,6 +397,63 @@ object HoodieSparkSqlWriter {
     (syncHiveSuccess, common.util.Option.ofNullable(instantTime))
   }
 
+  def bulkInsertAppendOnlyAsRow(sqlContext: SQLContext,
```

Comment:
bulkInsertAsRow and bulkInsertAppendOnlyAsRow has lot of common code. I am yet to unify them.